types/nightwatch/index.d.ts

```diff
@@ -904,127 +904,95 @@ export interface NightwatchAssertionsError {
     stack: string;
 }
 
-export interface NightwatchLanguageChains {
-    to: Expect;
-    be: Expect;
-    been: Expect;
-    is: Expect;
-    that: Expect;
-    which: Expect;
-    and: Expect;
-    has: Expect;
-    have: Expect;
-    with: Expect;
-    at: Expect;
-    does: Expect;
-    of: Expect;
-    include(...data: any): Expect;
-}
-
-export interface NightwatchExpectMethods {
-    active: Expect;
-    deep: Expect;
-    count: Expect;
+export interface NightwatchLanguageChains<T> {
```

Comment:
should this have default to 'Expect' to be backward compatible (sorry, if lame question, I'm not user of this specific module)

Reply:
From the user's perspective, nothing would change. It's just we are making sure of fine-grain control of types